ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterAggregator.java

```diff
@@ -39,28 +41,37 @@
   }
 
   /**
-   * Take a {@link List} of JSON string representation of {@link StatsWrapper} objects and perform cluster wide
+   * Take a {@link Map} of instance name to JSON string representation of {@link StatsWrapper} objects and perform cluster wide
    * aggregation with them.
-   * @param statsWrappersJSON a {@link List} of JSON string representation of {@link StatsWrapper} objects from the
+   * @param statsWrappersJSON a {@link Map} of instance name to JSON string representation of {@link StatsWrapper} objects from the
```

Comment:
how important is logging the instance? Seems like the change to this class and signatures of methods are all purely for logging the instance name. Is that not obtainable in some other way? Where does this object run?